src/LiveApi.js

```diff
@@ -125,10 +130,7 @@ export default class LiveApi {
     }
 
     onClose() {
-        setTimeout(() => {
-            this.connect();
-            this.resubscribe();
-        }, 1000);
+        this.connect();
```

Comment:
Was intentional :)
1sec timeout on disconnect.
On the other hand, it was totally not tested in real scenario, and if no delay works too, is better for sure ;)